### ISSUE TITLE
Small URL-Issue with Text Passages from Register.xml

### ISSUE THREAD

**Issue by BOberreither** [semantic-kraus/dw-data#53]
Just found out:
If a citedRange has a ref @target="[URL]" and @type="ext", then this has to be turned into an identifier. And it does, just: It adds all identifiers of all citedRange elements within the bibl to the first text passage.

E.g., "[Torquato Tasso](https://semantickraus.acdh.oeaw.ac.at/resource/?uri=https%3A%2F%2Fsk.acdh.oeaw.ac.at%2FDWbibl00013)" has two text passages, [the first](https://semantickraus.acdh.oeaw.ac.at/resource/?uri=https%3A%2F%2Fsk.acdh.oeaw.ac.at%2FDWbibl00013%2Fpassage%2F0) having two URL-Identifiers (only one of them correct), [the second one](https://semantickraus.acdh.oeaw.ac.at/resource/?uri=https%3A%2F%2Fsk.acdh.oeaw.ac.at%2FDWbibl00013%2Fpassage%2F1) has no URL-Identifier. (Just scroll down ti "Identifiers").

**Comment by BergListe** [comment]
@BOberreither 
These two paragraphs create the faulty identifiers:

````xml
#E42 url type identifier
<https://sk.acdh.oeaw.ac.at/DWbibl01058/identifier/idno/1> a cidoc:E42_Identifier ;
  rdfs:label "Identifier: https://textgridrep.org/browse/-/browse/11d2m_0#tg1224.2.169"@en ;
  cidoc:P2_has_type <https://sk.acdh.oeaw.ac.at/types/idno/URL/textgrid> ;
  cidoc:P1i_identifies <https://sk.acdh.oeaw.ac.at/DWbibl00013/passage/0> ;
  rdf:value "https://textgridrep.org/browse/-/browse/11d2m_0#tg1224.2.169" .

#E42 url type identifier
<https://sk.acdh.oeaw.ac.at/DWbibl01059/identifier/idno/1> a cidoc:E42_Identifier ;
  rdfs:label "Identifier: https://textgridrep.org/browse/-/browse/11d2m_0#tg1202.2.284"@en ;
  cidoc:P2_has_type <https://sk.acdh.oeaw.ac.at/types/idno/URL/textgrid> ;
  cidoc:P1i_identifies <https://sk.acdh.oeaw.ac.at/DWbibl00013/passage/0> ;
  rdf:value "https://textgridrep.org/browse/-/browse/11d2m_0#tg1202.2.284" .
````

I corrected the problem now but I'm wondering if the line about the passage is correct in the second paragraph: 
```` xml
 cidoc:P1i_identifies <https://sk.acdh.oeaw.ac.at/DWbibl00013/passage/0> ;
```` 

Could it be that this passage should be identifed as `passage/1` as it is the second citedRange in `DWbibl00013`?

**Comment by BOberreither** [comment]
@BergListe Yes, seems like this is where the actual error lies. (Keeping in mind that counting the citedRange elements always came with some caveats ...)

**Comment by BergListe** [comment]
@BOberreither i think i solved the problem. the updated xsl file is already in the git repo! please have a look and tell me if this fixes the issue!
